site/docs/guides/connect-network.md

```diff
@@ -0,0 +1,75 @@
+# Configure connections with SSH tunneling
+
+Depending on your enterprise network security, you may need to use [SHH tunneling](https://www.ssh.com/academy/ssh/tunneling/example#local-forwarding), or port forwarding, to allow Flow
+to securely access your endpoint. This is configured within a capture or materialization definition, but
+before you can do this, you'll need a properly configured SSH server on your internal network.
+
+These steps provide a basic roadmap, and should be followed with the help of an IT specialist in your organization,
+as details depend on your organization's policies and practices.
+
+1. Activate an [SSH implementation on a server](https://www.ssh.com/academy/ssh/server#availability-of-ssh-servers), if you don't have one already.
+Consult the documentation for your server's operating system and/or cloud service provider, as the steps will vary.
+Configure the server to your organization's standards, or reference the [SSH documentation](https://www.ssh.com/academy/ssh/sshd_config) for
+basic configuration options.
+
+2. Referencing the config files and shell output, collect the following information:
+
+   * The **endpoint** for the SSH server, which may look like the any of following:
+     * `ec2-198-21-98-1.compute-1.amazonaws.com`
+     * `198.21.98.1`
+     * `198.21.98.1:22`
```

Comment:
Same as the comments above, we are these endpoint examples could be changed into.
     * `ssh://ec2-198-21-98-1.compute-1.amazonaws.com`
     * `ssh://198.21.98.1`
     * `ssh://198.21.98.1:22`